bower.json

```diff
@@ -12,21 +12,22 @@
     "output"
   ],
   "dependencies": {
-    "purescript-prelude": "~0.1.3",
-    "purescript-eff": "~0.1.2",
-    "purescript-argonaut-core": "~0.2.3",
-    "purescript-argonaut-codecs": "~0.6.1",
-    "purescript-maybe": "~0.3.5",
-    "purescript-monoid": "~0.3.2",
-    "purescript-tuples": "~0.4.0",
-    "purescript-arrays": "~0.4.4",
-    "purescript-strings": "~0.7.1",
-    "purescript-sets": "~0.5.5",
-    "purescript-foldable-traversable": "~0.4.2",
-    "purescript-either": "~0.2.3",
-    "purescript-ansi": "~0.1.3",
-    "purescript-node-path": "~0.4.1",
-    "purescript-control": "~0.3.2",
-    "purescript-console": "~0.1.1"
+    "purescript-prelude": "~1.0.1",
+    "purescript-eff": "~1.0.0",
+    "purescript-argonaut-core": "~1.0.0",
+    "purescript-argonaut-codecs": "~1.0.0",
+    "purescript-maybe": "~1.0.0",
+    "purescript-monoid": "~1.0.0",
+    "purescript-tuples": "~1.0.0",
+    "purescript-arrays": "~1.0.0",
+    "purescript-strings": "~1.0.0",
+    "purescript-sets": "~1.0.0",
+    "purescript-foldable-traversable": "~1.0.0",
+    "purescript-either": "~1.0.0",
+    "purescript-ansi": "~1.0.0",
+    "purescript-node-path": "~1.0.0",
+    "purescript-control": "~1.0.0",
+    "purescript-console": "~1.0.0",
+    "purescript-now": "^1.0.0"
```

Comment:
What should? `purescript-now`? I cleaned both bower components dirs, ran `npm run setup` and `npm run build` and it builds cleanly.